benchmark/solver/solver.cpp

```diff
@@ -401,17 +463,15 @@ void solve_system(const char *solver_name,
             std::chrono::duration_cast<std::chrono::nanoseconds>(tac - tic);
```

Comment:
Missing `tac` ?

Reply:
It's on line [398](https://github.com/ginkgo-project/ginkgo/pull/172/files#diff-683717de88ad01033aa64401c6f87e94L398).